Bottom: cbcc5594a39e9b709b2d5eade559c917213b9aaa
Top:    ae20b6c33253556762ae92b1d0adedd8600f0a94
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 17:43:25 +0100

Refresh of draw-c-use-the-via-callback-wh

---

diff --git a/src/draw.c b/src/draw.c
index 8178e81..f204fdc 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -386,7 +386,7 @@ DrawEverything (BoxTypePtr drawn_area)
 		continue;
 	      r_search (PCB->Data->pin_tree, drawn_area, NULL, pin_callback,
 			NULL);
-	      r_search (PCB->Data->via_tree, drawn_area, NULL, pin_callback,
+	      r_search (PCB->Data->via_tree, drawn_area, NULL, via_callback,
 			NULL);
 	      /* draw element pads */
 	      if (group == component || group == solder)
